db/schema.rb

```diff
@@ -41,20 +41,6 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "flaggings", force: :cascade do |t|
```

Comment:
did you do this? while we're not using this table yet, i'm not sure it makes sense to delete